packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -217,6 +223,140 @@ describe('Checkout', () => {
                 // Assert
                 expect(addressBlock.exists()).toBe(false);
             });
+
+            it('should not display the address block if set to `dinein`', async () => {
+                // Act
+                const wrapper = shallowMount(VueCheckout, {
+                    store: createStore({ ...defaultCheckoutState, serviceType: CHECKOUT_METHOD_DINEIN }),
+                    i18n,
+                    localVue,
+                    propsData
+                });
+
+                const addressBlock = wrapper.find('[data-test-id="address-block"]');
+
+                // Assert
+                expect(addressBlock.exists()).toBe(false);
+            });
+
+            it('should display the table identifier input if set to `dinein`', async () => {
+                // Act
+                const wrapper = mount(VueCheckout, {
+                    store: createStore({ ...defaultCheckoutState, serviceType: CHECKOUT_METHOD_DINEIN }),
+                    i18n,
+                    localVue,
+                    propsData
+                });
+
+                const tableIdentifierInput = wrapper.find('[data-test-id="formfield-table-identifier-input"]');
```

Comment:
Move `'[data-test-id="formfield-table-identifier-input"]'` to a `const` ?

Reply:
I really want to change it but then awaiting 2 reviews again. 😂

Reply:
I feel your pain!